Yoast/wordpress-seo
# Implemented the extracted version of the Algolia Search

## Summary

This PR can be summarized in the following changelog entry:
Implements the extracted version of the Algolia Search which is now present in `yoast-components`.

## Test instructions

This PR can be tested by following these steps:

* If the PR on the `yoast-components` repository has not yet been merged, checkout the proper branch in that repository and use `yarn link` to link said branch to the Yoast SEO plugin.

* Go to a new post or any other page that introduces the Help Center.
* Go to the Knowledge Base tab.
* Verify that the Search works as it did before.

Requires https://github.com/Yoast/yoast-components/pull/149
Fixes #7223
